examples/multi-threaded/src/back.rs

```diff
@@ -41,9 +42,10 @@ pub async fn polar_init(polar: &mut PolarSensor, rx: watch::Receiver<bool>) -> P
             _ => {}
         }
     }
+    println!("Connected");
 
     if let Err(why) = polar.subscribe(arctic::NotifyStream::MeasurementData).await {
-        eprintln!("Could not subscirbe to measuremnt notifications: {:?}", why)
+        eprintln!("Could not subscribe to measurment notifications: {:?}", why)
```

Comment:
There's just one typo still, `measurment` -> `measurement`.

I can also just change this after merge if you want. Rest looks fine.